be/src/formats/orc/apache-orc/c++/src/Statistics.hh

```diff
@@ -572,7 +572,8 @@ public:
 
     void reset() override {
         _stats.reset();
-        setSum(Decimal());
+        // Do not compute sum of decimal column
+        // setSum(Decimal());
```

Comment:
You can't comment this line, it will failed in ut

Reply:
why do you care about a not-running UT?

Reply:
better not to change this code if it's not related.

apache orc has a bunch of UTs. it's not running now.

Reply:
The computation of sum of decimal values costs time. And we don't need it. That's why this is removed.

Reply:
no need to remove since it's not removed in apache repo https://github.com/apache/orc/blob/main/c%2B%2B/src/Statistics.hh#L705

Reply:
Have tested that orc writer of trino does not compute the sum of decimal either.

![Uploading mB5p9wR5dE.png…]()